modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/podcast/SubscribeManager.kt

```diff
@@ -187,6 +187,13 @@ class SubscribeManager @Inject constructor(
                 podcast.isSubscribed = subscribed
                 podcast.grouping = settings.podcastGroupingDefault.value
                 podcast.showArchived = settings.showArchivedDefault.value
+                if (FeatureFlag.isEnabled(Feature.CUSTOM_PLAYBACK_SETTINGS)) {
+                    podcastDao.findByUuid(podcastUuid)?.let { localPodcast ->
+                        podcast.copyPlaybackEffects(
+                            sourcePodcast = localPodcast,
+                        )
+                    }
+                }
```

Comment:
I'm copying local playback effects to the podcast here so that local playback effects are not lost while subscribing to it.